impl/src/main/java/org/ehcache/impl/persistence/DefaultLocalPersistenceService.java

```diff
@@ -17,21 +17,22 @@
 package org.ehcache.impl.persistence;
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-
+import org.ehcache.CachePersistenceException;
```

Comment:
Indeed, please configure the IDE so that it does not reorganise stuff you did not change - or adapt the rules so they mostly match what exists.